src/CarbonAware.WebApi/src/Models/EmissionsForecastBatchDTO.cs

```diff
@@ -0,0 +1,14 @@
+namespace CarbonAware.WebApi.Models;
+
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+[Serializable]
+public record EmissionsForecastBatchDTO : EmissionsForecastBaseDTO
+{
+  /// <summary>The historical time used to fetch the most recent forecast as of that time.</summary>
+  /// <example>2022-06-01T00:00:00Z</example>
+  [JsonPropertyName("requestedAt")]
+  [Required()]
+  public DateTimeOffset RequestedAt { get; set; }
```

Comment:
This will return the same historical forecast as if you had called `forecasts/current` at `2022-06-01T:00:00:03Z`.  For WattTime that would likely result in a forecast generatedAt `2022-06-01T:00:00:00Z`.

For hourly forecasts it would work similarly.  `requestedAt: 2022-06-01T:00:59:55Z` would grab the most recent forecast available at that time (also `generatedAt: 2022-06-01T:00:00:00Z`)

Reply:
Updated the example value to help make that point.